flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -618,19 +600,8 @@ public void releaseSnapshotResources() {
 				IOUtils.closeQuietly(readOptions);
 				readOptions = null;
 			}
-		}
 
-		/**
-		 * Drop the created snapshot if we have ben cancelled.
-		 */
-		public void dropSnapshotResult() {
-			if (null != snapshotResultStateHandle) {
-				try {
-					snapshotResultStateHandle.discardState();
```

Comment:
Cleanup is now handled somewhere else?

Reply:
Yes, the method was already unused before. Cleanup either happens already in the plain `close()` method of the checkpoint output stream, or in the `AsyncCheckpointRunnable::cleanup()` if ownership of a complete state handle was already transferred from the backend to the runnable.